auto_tag/README.md

```diff
@@ -0,0 +1,30 @@
+# Template for adding tag in an Azure Pipelines 
+
+Tags are references that point to specific points in Git history. Tagging is generally used to capture a point in history that is used for a marked version release (i.e. v1.0.1).
+With this template, tags created are of the format `<BuildId>-<Timestamp>` on top of the git commit, which is used for the main pipeline build. BuildId and CommitId paramters are sent from the main pipeline which is using this template.  
+
+## Usage
+
+Reference this repository inside the yaml pipeline
+
+```yaml
+resources:
+  repositories: 
+    - repository: UKHOTemplates
+      type: github
+      endpoint: GitHub_Service_Connection_Name
+      name: UKHO/devops-pipelinetemplates
+```
+
+Steps to add tagging template to your pipeline as job
```

Comment:
We can put something in the readme re. sensible usages? In our use case it's going to be a post-Live environment step so we can easily see in GitHub what the last commit that got to production was, so we don't anticipate there being vast numbers

Reply:
Since we are using timestamp in the tag, it would create multiple tags on the same commit if re-run this job. If needed we can add a task to delete existing tags before applying new tag. 
We did discuss about this among team and decided that it is not required as we seldom rerun the pipeline multiple times on live. Please let us know if it is better to remove existing tags

Reply:
Usage notes has been added